### ISSUE TITLE
Fix ID Translation on the Cheat Sheet and Add It to the Translations List

### ISSUE THREAD

**Issue by wisn** [github/training-kit#565]
## Overview
**TL;DR**
I've saw an effort to translate the cheat sheet to Bahasa Indonesia (see #403). However it's merged although unfinished. I do love to fix the translation and add it to the translations list on the cheat sheets page.

### How to Contribute
- [x] Comment on this issue if you'd like to work/collaborate on it! 
- [x] Fork the [repository](https://github.com/github/training-kit).
- [x] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com{{site.baseurl}}/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [x] Push your changes to your repository. 
- [x] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [x] Run `script/setup`
  - [x] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000{{site.baseurl}}`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by wisn** [comment]
I'm working on this right now :)

**Comment by zalew16** [comment]
#403
